config/sync/ultimate_cron.job.stanford_migrate_su_stanford_person.yml

```diff
@@ -4,7 +4,7 @@ status: true
 dependencies:
   module:
     - stanford_migrate
-title: 'Importer: Stanford Person Importer'
+title: 'Importer: Stanford Person'
```

Comment:
```suggestion
title: 'Importer: Person'
```

We don't have the word Stanford on the content types. Might as well remove it here as well.

Reply:
![semantics](https://media1.tenor.com/images/b7410780391e75251c41783f50cf2a0c/tenor.gif?itemid=15275994)